NewHorizons/Builder/Body/HeightMapBuilder.cs

```diff
@@ -91,14 +91,14 @@ public static GameObject Make(GameObject planetGO, Sector sector, HeightMapModul
                 level2.name += "1";
 
                 LODGroup.RecalculateBounds();
+
```

Comment:
Add a comment here saying that we only do the group stuff if LOD is on bc the only times LOD is off is for title screen and proxies